pallets/ocex/src/benchmarking.rs

```diff
@@ -400,26 +376,6 @@ benchmarks! {
 	set_snapshot {
 		let call = Call::<T>::set_snapshot{ new_snapshot_id: u64::MAX };
 	}: { call.dispatch_bypass_filter(RawOrigin::Root.into())? }
-
-	change_pending_withdrawal_limit {
-		let origin = T::GovernanceOrigin::try_successful_origin().unwrap();
-		let call = Call::<T>::change_pending_withdrawal_limit { new_pending_withdrawals_limit: u64::MAX };
-	}: { call.dispatch_bypass_filter(origin)? }
-
-	change_snapshot_interval_block {
-		let origin = T::GovernanceOrigin::try_successful_origin().unwrap();
-		let new_snapshot_interval_block = T::BlockNumber::decode(&mut 123u64.to_le_bytes().as_ref()).unwrap();
-		let call = Call::<T>::change_snapshot_interval_block{ new_snapshot_interval_block };
-	}: { call.dispatch_bypass_filter(origin)? }
-
-	whitelist_orderbook_operator {
-		let origin = T::GovernanceOrigin::try_successful_origin().unwrap();
-		let operator_public_key = sp_core::ecdsa::Public([u8::MAX; 33]);
-		let call = Call::<T>::whitelist_orderbook_operator { operator_public_key };
-	}: { call.dispatch_bypass_filter(origin)? }
-	verify {
-		assert!(<OrderbookOperatorPublicKey<T>>::get().unwrap() == operator_public_key);
-	}
```

Comment:
Extrinsic still [exists and tries to use weight](https://github.com/Polkadex-Substrate/Polkadex/blob/843169169b0400c279d380eefbbe0437955faa8d/pallets/ocex/src/lib.rs#L1048-L1057)  generated by this benchmark.

Reply:
reverted